GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -47,11 +47,15 @@ public string Prefetch(string args, bool failOnError)
             return this.CallGVFS("prefetch \"" + this.enlistmentRoot + "\" " + args, failOnError);
         }
 
-        public void Repair()
+        public void Repair(bool confirm)
         {
-            this.CallGVFS(
-                "repair --confirm \"" + this.enlistmentRoot + "\"", 
-                failOnError: true);
+            string confirmArg = confirm == true ? "--confirm " : string.Empty;
+            if (confirm)

```

Comment:
I don't think you should be checking this bool otherwise the repair will not be ran when `false`.  We still want to run it just without the `--confirm`